mentorship-platform/css/style.css

```diff
@@ -31,3 +31,34 @@
 }
 
 /*# sourceMappingURL=style.css.map */
+
+/*# Footer */
+.footer {
+  display: flex; 
+  justify-content: space-between;
+  flex-wrap: wrap; 
+  flex-direction: row;
+}
+@media (max-width: 580px) {
+  .footer {
+    flex-direction: column;
+  }
+}
+.footer__social-media-icons{
```

Comment:
.footer__social-media-icons this can be written inside the .footer class like .footer {
&__social-media-icons {
...
}

}